gdpr/impl.go

```diff
@@ -160,28 +170,11 @@ func (p *permissionsImpl) allowActivities(ctx context.Context, vendorID uint16,
 	return
 }
 
-func (p *permissionsImpl) isVendorException(purpose consentconstants.Purpose, bidder openrtb_ext.BidderName) (vendorException bool) {
-	if _, ok := p.purposeConfigs[purpose].VendorExceptionMap[bidder]; ok {
-		vendorException = true
-	}
-	return
-}
-
-func (p *permissionsImpl) isSpecialFeatureVendorException(bidder openrtb_ext.BidderName) (vendorException bool) {
-	if _, ok := p.cfg.TCF2.SpecialFeature1.VendorExceptionMap[bidder]; ok {
-		vendorException = true
-	}
-	return
-}
-
```

Comment:
These functions have been moved onto the `TCF2ConfigReader` implementations `config.GDPR.TCF2` and the aggregated config.